packages/content/src/requests/items/readAttribute/ReadAttributeAcceptResponseItem.ts

```diff
@@ -24,7 +24,7 @@ export class ReadAttributeAcceptResponseItem extends AcceptResponseItem implemen
     @validate()
     public attribute: IdentityAttribute | RelationshipAttribute;
 
-    public static override from(value: IReadAttributeAcceptResponseItem | ReadAttributeAcceptResponseItemJSON): ReadAttributeAcceptResponseItem {
+    public static override from(value: IReadAttributeAcceptResponseItem | Omit<ReadAttributeAcceptResponseItemJSON, "@type">): ReadAttributeAcceptResponseItem {
```

Comment:
you should still allow to add `@type`, so `IX | Omit<XJSON, "@type"> | XJSON` would be better here